### PR TITLE
[master] Updates `master` for WrenSec repos & info

### PR DESCRIPTION
**Pull request by Kortanul**
the changes here were already made for 8.6.0 (which is, at the moment, the only version we have needed for FR commons).

### CODE REVIEW

File: forgerock-ui-commons/pom.xml [hunk 1]
```diff
@@ -1,44 +1,45 @@
...
-  ~ Header Notice in each file and include the License file
-  ~ at http://forgerock.org/license/CDDLv1.0.html
-  ~ If applicable, add the following below the CDDL Header,
-  ~ with the fields enclosed by brackets [] replaced by
-  ~ your own identifying information:
-  ~ "Portions Copyrighted [year] [name of copyright owner]"
-  -->
+<!-- ~ DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+     ~
+     ~ Copyright (c) 2011-2013 ForgeRock AS. All Rights Reserved
```
Comment (siepkes) [inline forgerock-ui-commons/pom.xml]: The year in the FR license seems to change? `2011-2016` becomes `2011-2013`.

File: forgerock-ui-commons/pom.xml [hunk 2]
```diff
@@ -1,44 +1,45 @@
...
+     ~ Portions Copyright 2017 Wren Security.
```
Comment (siepkes) [inline forgerock-ui-commons/pom.xml]: Shouldn't these copyrights be at the bottom of the license (like elsewhere)?

File: forgerock-ui-commons/pom.xml (hunk 1 above)
Comment (Kortanul) [reply forgerock-ui-commons/pom.xml]: good catch, @siepkes. this came from forward-porting a fix to the license formatting from the 8.6.0 branch. I've now restored the copy on this branch.

File: forgerock-ui-commons/pom.xml (hunk 2 above)
Comment (Kortanul) [reply forgerock-ui-commons/pom.xml]: should now be fixed (per my message above).

**Comment by Kortanul** [comment]
@siepkes This got updated. Can you take a second look?

**Pull request merged**